src/main/docs/guide/appendix/architecture/containerArch.adoc

```diff
@@ -0,0 +1,30 @@
+Once the job of the <<compilerArch, Micronaut Compiler>> is complete and the required classes generated, it is up to the api:inject.BeanContext[] to load the classes for runtime execution.
```

Comment:
todo: add a section about `ContextConfigurer` once this isn't experimental anymore